app/components/deputy_assignment_component.html.erb

```diff
@@ -1,6 +1,10 @@
 <div id="<%= dom_id deputy_assignment %>" class="<%= root_class %>">
   <div class="deputy-assignment__name">
-    <strong><%= name %></strong> <%= webaccess_id %>
+    <% if current_user_is_deputy? && !pending? %>
+      <%= link_to name, profile_path(primary.webaccess_id) %> <%= webaccess_id %>
+    <% else %>
+      <strong><%= name %></strong> <%= webaccess_id %>
```

Comment:
This entire block could also be done as...

```
<strong><%= link_to_if (current_user_is_deputy? && confirmed?), name, profile_path(primary.webaccess_id) %></strong> <%= webaccess_id %>
```

It's a little more DRY but also a little more terse. Just thought I'd throw it out there.